extensions/binding/org.eclipse.smarthome.binding.sonos/src/main/java/org/eclipse/smarthome/binding/sonos/discovery/ZonePlayerDiscoveryParticipant.java

```diff
@@ -51,11 +51,12 @@ public DiscoveryResult createResult(RemoteDevice device) {
                 } catch (Exception e) {
                     // ignore and use default label
                 }
+                label += " (" + roomName + ")";
```

Comment:
This one is interesting.
The room name used to be the "representation property" as it is the information which is useful to the user to identify the device. The new UDN is really just something very technical, which won't be helpful to the user at all :-(
But I guess this is the side effect of the new auto-ignore feature - not much we can do about it.